components/authentication-framework/org.wso2.carbon.identity.application.authentication.framework/src/main/java/org/wso2/carbon/identity/application/authentication/framework/store/SessionDataStore.java

```diff
@@ -332,23 +333,23 @@ public void removeExpiredSessionData(Timestamp timestamp) {
         }
         try {
             statement = connection.prepareStatement(sqlDeleteExpiredDataTask);
-            long currentStandardNano = getCurrentStandardNano(timestamp);
-            statement.setLong(1, currentStandardNano);
+            long cleanupLimit = FrameworkUtils.getCurrentStandardNano() - IdentityUtil.getCleanUpTimeout() * 60 * 10^9;
```

Comment:
Use `TimeUnit.SECONDS. toNanos(IdentityUtil.getCleanUpTimeout())` for better readability of the code.

And we can have variable name as `cleanupLimitNano` for the same purpose.